src/storage/rpc.rs

```diff
@@ -106,8 +106,8 @@ impl Storage for RpcStorage {
             let basic = AccountBasic {
                 balance,
                 nonce,
-                code_hash: Some(code.hash_slow()),
-                code: Some(code.into()),
+                code_hash: (!code.is_empty()).then(|| code.hash_slow()),
```

Comment:
This isn't very clean, but I also want to move `code` out of `AccountBasic` so let's go with it for now.